### DISCUSSION TITLE
Question - Auto Import Not Running

### DISCUSSION THREAD

**Discussion by jsholcomb** [photoprism/photoprism#4345] [General]
I had a question and had searched a few places and was not seeing any other reports of this issue so I was concerned it might be an isolated instance.

I am running Photoprism in Docker on a Win 11 Machine which has been fantastic for months.  Recently I upgraded the instance to the latest release version and have noticed photos being synced by Photosync are successfully being uploaded, but are not being imported by the auto import.

I can use the GUI to go to Library - Import, and click Import to manually start this.  Once I manually start the import, it successfully identifies the new files and processes them.  It then loads those files into my library however its a pure manual process to get the import to trigger.

Any ideas what might be causing this?

**Comment by lastzero** [comment]
With our latest release, you need to explicitly enable this feature:
- https://github.com/photoprism/photoprism/issues/4310#issuecomment-2143824192

**Reply by jsholcomb** [reply]
Thank you very much for that clarification, that is perfect.
